test/hotspot/jtreg/vmTestbase/nsk/jdi/ThreadReference/stop/stop002.java

```diff
@@ -115,7 +115,10 @@ private int runIt(String args[], PrintStream out) {
             return quitDebuggee();
         }
 
-        ThreadReference thrRef = debuggee.threadByName(DEBUGGEE_THRNAME);
+        // debuggee main class
+        mainClass = debuggee.classByName(DEBUGGEE_CLASS);
```

Comment:
Maybe just move this into `try` and replace `debuggee.threadByFieldName` with `debuggee.threadByFieldNameOrThrow`?

Reply:
@alexmenkov It looks like this could be done in almost every test in this PR. In some the `try` just needs to move about 8 lines up, but in others it's more like 20. There will be a lot of indentation diffs. Do you still want to see this change?

Reply:
It looks like there were some changes in part 3 that could also benefit from this. Perhaps this would be better off done in a followup PR. It will keep this PR easier to review, will make moving the `try` easier to review, and will pickup some additional tests that could use this.

Reply:
I field [JDK-8357172](https://bugs.openjdk.org/browse/JDK-8357172)